lib/launchers/openstack4.rb

```diff
@@ -66,6 +68,10 @@ def initialize(**options)
       self.get_token()
     end
 
+    def proxy_set?()
+        return @proxy ? true : false
+    end
+
```

Comment:
Is openstack4.rb still used ?

Reply:
no idea, but since it's OpenSourced, so better with it.

Reply:
we need to remove openstack4

Reply:
but it is fine to keep it for the time being. Just saying it is not really needed